examples/simple-project/_github/workflow.yml

```diff
@@ -0,0 +1,43 @@
+# This GitHub Actions workflow triggers after receiving a Deployment Status event from GH Actions environment. This works
+# with providers like Vercel, Heroku and others. This workflow does the following
+#
+# - Runs after deployment event
+# - Uses the target_url provided in the deployment event
+# - Dry run all the checks on Checkly
+# - If the dry run passes, deploy the checks.
+#
+#  You can of course trigger the Checkly CLI in other workflows or steps inside your GH Action configuration.
+name: checkly
+on: [deployment_status]
+env:
+  ENVIRONMENT_URL: ${{ github.event.deployment_status.target_url }}
```

Comment:
I don't think that is correct. We gotta pass check envs with the `-e` flag

Reply:
@umutuzgur not in my example code, it already works with https://github.com/checkly/checkly-cli/blob/a611222b2917a4414f75be49b5a25d2e623283b6/examples/simple-project/src/defaults.js

Reply:
...or does it...

Reply:
I'm not sure if it works. I can guarantee it won't work with typescript given we have an in-memory transpiler in between

Reply:
I'm testing this right now

Reply:
@umutuzgur you were right, it's fixed now